test/org/impact2585/frc2016/tests/WheelSystemTest.java

```diff
@@ -139,6 +144,25 @@ public void test() {
 		drivetrain.setCurrentRampForward(0.5);
 		drivetrain.run();
 		Assert.assertTrue(currentRampForward == ramp && rotate == 0);
+		
+		//tests rotation exponent
+		toggleRotationExponent = true;
+		rotate = 0.4;
+		drivetrain.run();
+		Assert.assertTrue(currentRampForward == ramp && rotate == Math.pow(0.4, WheelSystem.SECONDARY_ROTATION_EXPONENT));
+		
+		//test if the rotation exponent doesn't toggle if the toggle is pressed
+		toggleRotationExponent = true;
+		rotate = 0.4;
+		drivetrain.run();
+		Assert.assertTrue(currentRampForward == ramp && rotate == Math.pow(0.4, WheelSystem.SECONDARY_ROTATION_EXPONENT));
+		
+		//tests if the rotation exponent can be toggled off
+		toggleRotationExponent = false;
+		drivetrain.run();
+		rotate = 0.5;
+		toggleRotationExponent = true;
+		Assert.assertTrue(currentRampForward == ramp && rotate == Math.pow(0.5, WheelSystem.PRIMARY_ROTATION_EXPONENT));
```

Comment:
It looks like you forgot to call `drivetrain.run()`. Also, the fact that the unit test is still passing with that seems like a bad sign.